alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
I understand that the directory for Alembic is created by default when running alembic init. But I think it would be good to try to create the Alembic environment in the directory created for the database and I see that it can be done by running the command like this: alembic init app/db/migrations